site/content/docs/developer/quick-start-build.md

```diff
@@ -170,4 +170,7 @@ cd ..
 ```
 
 ### Make Changes and Submit a PR!
+
+We welcome any changes made by a Contributor following a review process conducted by a ZAP core team member. Prior to submission of a PR, please be aware: Contributions must adhere to the [Guidelines](https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md#guidelines-for-pull-request-pr-submission-and-processing).
```

Comment:
I'd prefer if this didn't go into details saying how many core team members do the reviews, as is now it seems that a single reviewer is enough but as mentioned in the linked page it's usually required two.

Reply:
> We welcome any changes made by a Contributor following a review process. Prior to submission of a PR, please be aware: Contributions must adhere to the [Guidelines](https://github.com/zaproxy/zaproxy/blob/main/CONTRIBUTING.md#guidelines-for-pull-request-pr-submission-and-processing).